micro-s3/src/main/java/com/aol/micro/server/s3/S3Utils.java

```diff
@@ -54,4 +70,23 @@ public void delete(String bucketName, List<KeyVersion> objects) {
 		});
 	}
 
+	public InputStream getInputStream(String bucketName, String key, Supplier<File> tempFileSupplier)
```

Comment:
javadoc for the methods in S3Utils would also be good. 

Should this class be renamed to something like S3Access?